### DISCUSSION TITLE
Cannot see website after publishing

### DISCUSSION THREAD

**Discussion by mailnrubbish** [community/community#22707] [New to GitHub]
<p>Hi</p>
<p>I am  a brand new user and I have just created a new repository. I added the files from Explorer to the repository directory and pushed the files to github. However I cannot view them in a browser no matter what URL I enter - please can someone help</p>
<p>many thanks</p>
<p>Paul</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/mailnrubbish">@mailnrubbish</a>, welcome to the GitHub Support Community!</p>
<p>Your issue is related to the <a href="https://github.com/mailnrubbish/Test2" rel="nofollow noopener">Test2</a> and <a href="https://github.com/mailnrubbish/Test3" rel="nofollow noopener">Test3</a> repos, correct?</p>
<p>From what I can tell, it looks like the issue is related to the naming of your <strong>index.html</strong> files; the first letter is currently capitalized. The naming of files/directories on GitHub’s servers are case-sensitive, so changing the filename from <strong>I</strong>ndex.html to <strong>i</strong>ndex.html should fix the issue.</p>

**Comment by mailnrubbish** [comment]
<p>Many thanks for that - simple when you know how :smile:</p>
